pkg/kubelet/dockertools/docker.go

```diff
@@ -126,7 +126,12 @@ func (p dockerPuller) Pull(image string) error {
 		Tag:        tag,
 	}
 
-	creds, haveCredentials := p.keyring.Lookup(repoToPull)
+	keyring, err := credentialprovider.MakeDockerKeyring(secrets, p.keyring)
```

Comment:
@bgrant0607 Just need to add this sentence in rkt's Pull as well for docker images.